### DISCUSSION TITLE
Black-Owned Businesses in Seattle

### DISCUSSION THREAD

**Discussion by ebar23** [jakobzhao/geog495#67] [Final Project]
Link: https://ebar23.github.io/Black-OwnedBiz_Final_Project/index.html#
Github: https://github.com/ebar23/Black-OwnedBiz_Final_Project

Description: This project was intended to bring more awareness to the locations of Black-Owned Businesses in the Seattle area. While we still continue to live in a COVID present world where things are slowly but surely getting back to normal, businesses have been impacted greatly and are trying to stay afloat. Minority-owned businesses are no exception to this, and for this reason, I decided to dedicate my project to the African-American community here in Seattle to help them gain more clientele. 

This map will list 81 restaurants and will show the nearest restaurant to your location when you type it into the search bar.

![Screen Shot 2021-12-09 at 11 26 01 PM](https://user-images.githubusercontent.com/32309682/145537688-dd69f329-19bc-4606-8314-d6fe80f9abe0.png)

**Comment by seanhy1** [comment]
I really love the choice of topic here. One specific feature of your application that deserves praise is the choice of color scheme, associating it to BLM colors (red, green, black). Your fly-to functionality works smoothly as I interact from one business to another. Very well done!

One change I suggest making is figuring out a way to resolve the text conflicts occurring in your side-panel (if possible). To elaborate on what I mean by that, is that for every business that has an apostrophe in their name, the side-panel displays a '?' symbol in replacement of the " ' " character. With that being said, try to find a way to tackle this issue if you can. It would make your web application design look much more polished in my opinion, but it is only a minimal change to be made. Nonetheless, great job! This web application clearly carries meaning and significance to the project's goal.

**Comment by TheAdonalsium** [comment]
As some have said before, you went the extra step with your colors, choosing a very fitting palette for this assignment. I also really like the highlighting of what you're looking at on the sidebar. One thing I would suggest to improve is the ? icons in the sidebar, probably apostrophes that didn't make it through formatting. In my project I just directly removed them, but if you could figure out how to keep them in, that would likely be better overall.

**Comment by lucretiachen99** [comment]
I think this is a great idea of a project because I have not seen another person create a map like this before so it is very unique and supportive to the black community. Something that would help improve the map would be to use different icons or colors for different types of businesses such as restaurants or stores, etc.

**Comment by Anemnox** [comment]
This project does a good job in representing your goal for your description! It is really cool to see all the different black owned businesses in the area of Seattle and makes it possible to identify and support the businesses. One thing that would be cool is to filter the businesses by their category to make it possible to identify a restaurant or a store based on the user's goal.

**Comment by emilyphantastic** [comment]
Your project topic is wonderful!
I like your choice in colors and base map. Colors because it represents the "BLM colors" and base map because it's relevant to the application. You're able to see streets and roads nearby the restaurants. Something I would suggest is changing the title from "business" to "restaurant." This is because before reading your project description, I thought it was a map full of different black owned businesses and was wondering what service or product each location had. It wasn't until I read your description to realize it was all restaurants.

**Comment by meleakewubbie** [comment]
I really like the topic you chose. The base map you chose and the colors you used are aesthetically pleasing. I recommend making the side panel a little bit smaller if possible.
